pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -84,7 +84,7 @@ stdenv.mkDerivation (finalAttrs: {
     # not need to know binutils' BINDIR at all. It's an absolute path
     # where libraries are stored.
     ./plugins-no-BINDIR.patch
-  ] ++ lib.optionals buildPlatform.isDarwin [
+  ] ++ lib.optionals hostPlatform.isDarwin [
```

Comment:
This patch could be applied unconditionally, but otherwise the Darwin changes lgtm.